nrovalidate/main.go

```diff
@@ -59,6 +61,7 @@ type ProgArgs struct {
 	Verbose bool
 	Quiet   bool
 	JSON    bool
+	What    sets.String
```

Comment:
can we rename What to better word that reflects the use of the arg? maybe ValidatorType or similar

Reply:
sure